main/calculations.py

```diff
@@ -0,0 +1,147 @@
+import requests
+import json
+from datetime import datetime
+from constance import config
+from proxy_requests.proxy_requests import ProxyRequests
+
+from django.core.mail import send_mail
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib import messages
+
+from .models import SiteToCheck
+
+
+def send_email(message, email):
```

Comment:
Możesz używać czegoś takiego jak kwargs. Dzięki czemu nie musisz tworzyć sztucznych referencji `message = message`

Wyglądało by to tak:

```
def send_email(message, email):
    send_mail(
        subject='Errors on my pages',
        message=message,
        from_email=settings.EMAIL_ADDRESS,
        recipient_list=[email],
        fail_silently=False
    )
```